website/versioned_docs/version-experimental/RelayHooks-StepByStep.md

```diff
@@ -0,0 +1,300 @@
+
+
+## Step 4: Adding Relay To Our Project
+
+In this guide we'll demonstrate installing the *experimental* release of Relay Hooks, a new, hooks-based Relay API that supports React Concurrent Mode and Suspense. 
+
+First we'll add the necessary packages. Note that Relay is comprised of three key pieces: a compiler (which is used at build time), a core runtime (that is React agnostic), and a React integration layer.
+
+```bash
+# NPM Users
+npm install --save relay-runtime react-relay@experimental
+npm install --save-dev relay-compiler graphql
+
+# Yarn Users
+yarn add relay-runtime react-relay@experimental
+yarn add --dev relay-compiler graphql babel-plugin-relay
+```
+
+*Note*: You may get a notice asking you to choose which version of `relay-runtime` to use - if so, specify version 7.0.x (e.g. `7.0.0`).
+
+### 4.1 Configure Relay Compiler
+
+Next let's configure Relay compiler. We'll need a copy of the schema as a `.graphql` file. If you're using the GitHub GraphQL API, you can download a copy directly from the Relay example app:
+
+```bash
+cd your-app-name
+curl https://raw.githubusercontent.com/relayjs/relay-examples/master/issue-tracker/schema/schema.graphql > schema.graphql
+```
+
+If you're using your own API we suggest using the [`get-graphql-schema`](https://www.npmjs.com/package/get-graphql-schema) utility to download your schema into a `.graphql` file.
+
+Now that we have a schema we can modify `package.json` to run the compiler first whenever we build or start our app:
+
+```json
+// your-app-name/package.json
+{
+  ...
+  "scripts": {
+    ...
+    "start": "yarn run relay && react-scripts start",
+    "build": "yarn run relay && react-scripts build",
+    "relay": "yarn run relay-compiler --schema schema.graphql --src ./src/ --watchman false $@"
+    ...
+  },
+  ...
+}
+```
+
+At this point, you should be able to run the following successfully:
+
+```bash
+cd your-app-name
+yarn start
+```
+
+If it works, your app will open at [localhost:3000](http://localhost:3000). Now when we write GraphQL in our app, Relay will detect it and generate code to represent our queries in `your-app-name/src/__generated__/`. We recommend checking in these generated files to source control.
+
+### 4.2 Configure Relay Runtime 
```

Comment:
I would just say Relay here; I think what "runtime" means may not always be obvious for people, so it might help avoid confusion

Reply:
i mention the  compiler above